src/constants/ui.ts

```diff
@@ -1,2 +1,3 @@
 export const MIN_BUTTON_SIZE = 48
+export const TABLET_WIDTH_THRESHOLD = 767
```

Comment:
where do we use this constant?

Reply:
> where do we use this constant?

we actually only use this for the admin panel

Reply:
So you are going to work further on this branch, right? I think this code changes are too small to be a PR.

Reply:
> So you are going to work further on this branch, right? I think this code changes are too small to be a PR.

I don't think this would be necessary; closed it because admin is already merged so changes are already reflected there:)